api/test/testdata/import-test-plugins.yaml

```diff
@@ -0,0 +1,99 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# If you want to set the specified configuration value, you can set the new
+# in this file. For example if you want to specify the etcd address:
+#
+consumes:
+- application/json
+- application/xml
+components:
+  securitySchemes:
+    basicAuth:
+      type: http
+      scheme: basic
+      username: test
+      password: testp
+info:
+  description: |-
+    test desc
+  license:
+    name: Apache License 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+  title: |-
+    test title
+paths:
+  /hello:    
+    post:
+      x-api-limit: 20
+      description: |-
+        hello world.
+      operationId: hello
+      x-apisix-upstream:
+        type: roundrobin
+        nodes:
+          - host: "172.16.238.20"
+            port: 1980
+            weight: 1
+      parameters:
+      - name: id
+        in: header
+        description: ID of pet to use
+        required: true
+        schema:
+          type: array
+          items:
+            type: string
+        style: simple
+
+      requestBody:
+        content:
+          'application/x-www-form-urlencoded':
+            schema:
+              properties:
+                  name: 
+                    description: Updated name of the pet
+                    type: string
+                  status:
+                    description: Updated status of the pet
+                    type: string
+              required:
+                - status
+
+      produces:
+      - application/json
+
+      security:
+      - basicAuth: []
+
+      responses:
+        "0":
+          description: list response
+          schema:
+            items:
+              $ref: '#/definitions/service'
+            type: array
+        default:
+          description: unexpected error
+          schema:
+            $ref: '#/definitions/ApiError'
+produces:
+- application/json
+- application/xml
+schemes:
+- http
+- https
+swagger: "2.0"
```

Comment:
I noticed that  the version of this test file is swagger is 2.0, and our basic version of this issue is OAS3.0, which equals to swagger 3.0. As far as I know, there are some difference between the these versions, so I just want to make sure that is the current feature compatible with two versions of file or just for 2.0?

For the export feature in #1245 is only for 3.0 now, baseline alignment is necessary.

Reply:
fixed.